prime-react/src/main/scala/react/primereact/InputText.scala

```diff
@@ -16,6 +16,7 @@ case class InputText(
   value:       js.UndefOr[String] = js.undefined,
   disabled:    js.UndefOr[Boolean] = js.undefined,
   placeholder: js.UndefOr[String] = js.undefined,
+  onFocus:     js.UndefOr[ReactFocusEventFromInput => Callback] = js.undefined,
```

Comment:
From Prime's documentation (https://www.primefaces.org/primereact/inputtext/) it seems that `InputText` only has 4 original properties and the rest are passed through to the `input`.

Should we consider removing the properties that can be passed by `modifiers`? (Which seem to be all of the current ones)

Reply:
> Should we consider removing the properties that can be passed by `modifiers`? (Which seem to be all of the current ones)

Ah. So everything we pass with `.withMods` is passed on the input? Yeah, I guess we could remove all of them, then. I don't feel strongly either way. They were on the object ST created, so I used them.